dali/operators/reader/loader/file_label_loader.h

```diff
@@ -47,21 +47,47 @@ class DLL_PUBLIC FileLabelLoader : public Loader<CPUBackend, ImageLabelWrapper>
  public:
   explicit inline FileLabelLoader(
     const OpSpec& spec,
-    vector<std::pair<string, int>> image_label_pairs = std::vector<std::pair<string, int>>(),
     bool shuffle_after_epoch = false)
     : Loader<CPUBackend, ImageLabelWrapper>(spec),
-      file_root_(spec.GetArgument<string>("file_root")),
-      file_list_(),
-      image_label_pairs_(std::move(image_label_pairs)),
       shuffle_after_epoch_(shuffle_after_epoch),
       current_index_(0),
       current_epoch_(0) {
 
+      if (!spec.TryGetArgument(file_root_, "file_root") && !spec.HasArgument("files")) {
+        DALI_FAIL("``file_root`` argument is required when not supplygin explicit file paths "
```

Comment:
Is it? If it is not provided I would rather expect that the paths in the file_root_ are absolut.

Reply:
Why should I make life more difficult for the user?